src/OSPSuite.R/Services/ContainerTask.cs

```diff
@@ -139,15 +147,28 @@ public IParameter[] AllParametersForSensitivityAnalysisMatching(ISimulation simu
       public string[] BaseUnitNamesFromPath(IModelCoreSimulation simulation, string path) =>
          AllQuantitiesMatching(simulation, path).Select(x => x.BaseUnitName()).ToArray();
 
-      public string[] DimensionNamesFromPath(IModelCoreSimulation simulation, string path) => 
+      public string[] DimensionNamesFromPath(IModelCoreSimulation simulation, string path) =>
          AllQuantitiesMatching(simulation, path).Select(x => x.DimensionName()).ToArray();
 
-      public bool[] IsExplicitFormulasFromPath(IModelCoreSimulation simulation, string path) => 
+      public bool[] IsExplicitFormulasFromPath(IModelCoreSimulation simulation, string path) =>
          AllQuantitiesMatching(simulation, path).Select(x => x.Formula.IsExplicit()).ToArray();
 
-      public void AddQuantitiesToSimulationOutputFromPath(IModelCoreSimulation simulation, string path) => 
+      public void AddQuantitiesToSimulationOutputFromPath(IModelCoreSimulation simulation, string path) =>
          AllQuantitiesMatching(simulation, path).Each(simulation.OutputSelections.AddQuantity);
 
+      public void SetValueByPath(IModelCoreSimulation simulation, string path, double value)
+      {
+         if (path.Contains(WILD_CARD))
+            throw new OSPSuiteException(Error.CannotSetValueByPathUsingWildCard(path));
```

Comment:
@PavelBal I think this should be an error
Also when setting a value for a path that does not exist